src/util/agent_control.c

```diff
@@ -657,7 +657,7 @@ int main(int argc, char **argv)
             } else if (agent_id) {
                 printf("\n** Unable to run active response on all agents.\n");
             } else {
-                printf("\n** Unable to run active response on agent: %s\n", agent_id);
+                printf("\n** Unable to run active response on agent: NULL\n");
```

Comment:
This is a bug actually and should be reverted.

If you read the code you will see that `agent_id` is NULL when the AR targets all agents, otherwise is a specific agent. Current line aims to print an error when an AR went wrong in a single agent, thus L657 should be changed to ` } else if (NULL == agent_id) {`
```suggestion
                printf("\n** Unable to run active response on agent: %s\n", agent_id);
```